pkg/fleet/installer/service/apm_inject.go

```diff
@@ -62,12 +82,35 @@ type apmInjectorInstaller struct {
 	ldPreloadFileUninstrument *fileMutator
 	dockerConfigInstrument    *fileMutator
 	dockerConfigUninstrument  *fileMutator
+	envs                      *env.Env
 }
 
 // Setup sets up the APM injector
-func (a *apmInjectorInstaller) Setup(ctx context.Context) (retErr error) {
+func (a *apmInjectorInstaller) Setup(ctx context.Context) error {
+	err := a.Instrument(ctx)
```

Comment:
either set Instrument at the end or propagate the rollback, or you'll fail installation and still instrument!

Reply:
👍 Done